### ISSUE TITLE
Fibaro Walli Switch cannot set config

### ISSUE THREAD

**Issue by TerrorSource** [domoticz/domoticz#3557]
Hi,

I've written the OpenZwave config for the "FIBARO System FGWDS221 Walli Double Switch+" which is now merged into Domoticz.

When i wrote the config, i was able to set config's and apply them on the sensor. Now, it is not possible anymore with the latest version of Domoticz.

I also see config's i did not set in the OpenZwave config.
Config i wrote is on https://github.com/OpenZWave/open-zwave/pull/1869

config's which i did not wrote:
3001. Instance 1: Protection: Unprotected
Protect a device against unintentional control

3002. Instance 2: Protection: Unprotected
Protect a device against unintentional control

3003. Instance 3: Protection: Unprotected
Protect a device against unintentional control

**Comment by petergebruers** [comment]
Hi! Which parameter (configuration option) does not work? Can you please enable debug logging on the Z-Wave controller, then do "Refresh node info" + whatever config you are trying to change then zip and upload OZW_Log.txt? I'll have a look at it.

> Config i wrote is on [OpenZWave/open-zwave#1869](https://github.com/OpenZWave/open-zwave/pull/1869)

I remember, I merged the PR for you.

> I also see config's i did not set in the OpenZwave config.

The "protection" instances you see are not "parameters", they are a different command class and OpenZWave can obtain them by interviewing the device. So even though you did not write that in the config/fibaro/fgwds221.xml file, OZW discovers them. That is OK.

Parameters cannot be discovered (in general) and that's why you have to enter them in the config file.

**Comment by gizmocuz** [comment]
@TerrorSource, the process of getting new configuration files inside Domoticz from the OpenZWave repository is not an automatic job and needs to be done manually
But i recently merged the configuration files, and they should be available in beta 11083

**Comment by TerrorSource** [comment]
@petergebruers 

Here the requested data. Already removed the unnecessary lines from the log(working sensors)
 
013 (0x0d) | FW_Badkamer | FIBARO System FGWDS221 Walli Double Switch+ | FIBARO System | 0x1000 | 0x1b01 | 2019-08-12 08:50:06 | No

2019-08-12 08:48:39.384 Status: OpenZWave: Closed
2019-08-12 08:48:41.619 Status: OpenZWave: using config in: /home/pi/domoticz/Config/
2019-08-12 08:48:41.623 Status: OpenZWave: Starting...
2019-08-12 08:48:41.623 Status: OpenZWave: Version: 1.6-808-gbb0a4bf7-dirty
2019-08-12 08:48:42.027 OpenZWave: Value_Added: Node: 18 (0x12), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.027 OpenZWave: Value_Added: Node: 18 (0x12), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.027 OpenZWave: Value_Added: Node: 18 (0x12), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.028 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.028 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.028 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: SENSOR MULTILEVEL, Label: Illuminance, Instance: 1, Index: 3
2019-08-12 08:48:42.029 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.029 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.029 OpenZWave: Value_Added: Node: 19 (0x13), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.030 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.030 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.030 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: SENSOR MULTILEVEL, Label: Illuminance, Instance: 1, Index: 3
2019-08-12 08:48:42.031 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.031 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.031 OpenZWave: Value_Added: Node: 20 (0x14), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.032 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.032 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.032 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: Smoke Alarm, Instance: 1, Index: 1
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: System, Instance: 1, Index: 9
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.033 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.034 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: SENSOR ALARM, Label: General, Instance: 1, Index: 0
2019-08-12 08:48:42.034 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: SENSOR ALARM, Label: Smoke, Instance: 1, Index: 1
2019-08-12 08:48:42.034 OpenZWave: Value_Added: Node: 21 (0x15), CommandClass: SENSOR ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.035 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.035 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.035 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: Smoke Alarm, Instance: 1, Index: 1
2019-08-12 08:48:42.035 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.036 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.036 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.036 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: System, Instance: 1, Index: 9
2019-08-12 08:48:42.036 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.036 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.037 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: SENSOR ALARM, Label: General, Instance: 1, Index: 0
2019-08-12 08:48:42.037 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: SENSOR ALARM, Label: Smoke, Instance: 1, Index: 1
2019-08-12 08:48:42.037 OpenZWave: Value_Added: Node: 22 (0x16), CommandClass: SENSOR ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.038 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.038 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.038 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: Smoke Alarm, Instance: 1, Index: 1
2019-08-12 08:48:42.038 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.038 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.039 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.039 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: System, Instance: 1, Index: 9
2019-08-12 08:48:42.039 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.039 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.040 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: SENSOR ALARM, Label: General, Instance: 1, Index: 0
2019-08-12 08:48:42.040 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: SENSOR ALARM, Label: Smoke, Instance: 1, Index: 1
2019-08-12 08:48:42.040 OpenZWave: Value_Added: Node: 23 (0x17), CommandClass: SENSOR ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.040 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: SWITCH BINARY, Label: Switch, Instance: 1, Index: 0
2019-08-12 08:48:42.040 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: SENSOR MULTILEVEL, Label: Power, Instance: 1, Index: 4
2019-08-12 08:48:42.041 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: METER, Label: Electric - kWh, Instance: 1, Index: 0
2019-08-12 08:48:42.041 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: METER, Label: Electric - W, Instance: 1, Index: 2
2019-08-12 08:48:42.041 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: METER, Label: Exporting, Instance: 1, Index: 256
2019-08-12 08:48:42.041 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.041 OpenZWave: Value_Added: Node: 24 (0x18), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.042 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.043 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.043 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.043 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: ALARM, Label: Access Control, Instance: 1, Index: 6
2019-08-12 08:48:42.043 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.044 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.044 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.044 OpenZWave: Value_Added: Node: 25 (0x19), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.045 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.045 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.046 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.046 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: ALARM, Label: Access Control, Instance: 1, Index: 6
2019-08-12 08:48:42.046 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.046 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.046 OpenZWave: Value_Added: Node: 26 (0x1a), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.047 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.047 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.048 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.048 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: ALARM, Label: Access Control, Instance: 1, Index: 6
2019-08-12 08:48:42.048 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.048 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: ALARM, Label: Power Management, Instance: 1, Index: 8
2019-08-12 08:48:42.048 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.049 OpenZWave: Value_Added: Node: 27 (0x1b), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.049 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: SCENE ACTIVATION, Label: Scene, Instance: 1, Index: 0
2019-08-12 08:48:42.049 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: SCENE ACTIVATION, Label: Duration, Instance: 1, Index: 1
2019-08-12 08:48:42.049 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: SENSOR BINARY, Label: Sensor, Instance: 1, Index: 0
2019-08-12 08:48:42.050 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: SENSOR MULTILEVEL, Label: Air Temperature, Instance: 1, Index: 1
2019-08-12 08:48:42.050 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: ALARM, Label: Heat, Instance: 1, Index: 4
2019-08-12 08:48:42.051 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: ALARM, Label: Access Control, Instance: 1, Index: 6
2019-08-12 08:48:42.051 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: ALARM, Label: Home Security, Instance: 1, Index: 7
2019-08-12 08:48:42.051 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: ALARM, Label: System, Instance: 1, Index: 9
2019-08-12 08:48:42.051 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: ALARM, Label: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:48:42.051 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: BATTERY, Label: Battery Level, Instance: 1, Index: 0
2019-08-12 08:48:42.052 OpenZWave: Value_Added: Node: 28 (0x1c), CommandClass: SENSOR ALARM, Label: General, Instance: 1, Index: 0
2019-08-12 08:48:42.052 OpenZWave: Value_Added: Node: 29 (0x1d), CommandClass: SWITCH BINARY, Label: Switch, Instance: 1, Index: 0
2019-08-12 08:48:42.052 OpenZWave: Value_Added: Node: 29 (0x1d), CommandClass: SCENE ACTIVATION, Label: Scene, Instance: 1, Index: 0
2019-08-12 08:48:42.052 OpenZWave: Value_Added: Node: 29 (0x1d), CommandClass: SCENE ACTIVATION, Label: Duration, Instance: 1, Index: 1
2019-08-12 08:48:41.999 Status: OpenZWave: Driver Ready
2019-08-12 08:48:52.054 Status: OpenZWave: Node Removed. HomeID: 4100944663, NodeID: 13 (0x0d)
2019-08-12 08:49:04.878 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 6 (0x06)
2019-08-12 08:49:07.879 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 9 (0x09)
2019-08-12 08:49:19.835 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 13 (0x0d)
2019-08-12 08:49:42.785 (ZwaveUSB) General/kWh (Keuken_kWh Meter)
2019-08-12 08:49:42.845 (ZwaveUSB) Usage (Keuken_Unknown1)
2019-08-12 08:49:42.845 (ZwaveUSB) General/kWh (Keuken_kWh Meter)
2019-08-12 08:49:42.582 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 15 (0x0f)
2019-08-12 08:49:46.457 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: SWITCH BINARY, Label: Switch, Instance: 1, Index: 0
2019-08-12 08:49:46.459 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: CENTRAL SCENE, Label: Scene Count, Instance: 1, Index: 256
2019-08-12 08:49:46.107 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 24 (0x18)
2019-08-12 08:50:03.890 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: SWITCH BINARY, Label: Instance 2: Switch, Instance: 2, Index: 0
2019-08-12 08:50:03.891 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: SWITCH BINARY, Label: Instance 3: Switch, Instance: 3, Index: 0
2019-08-12 08:50:03.933 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 1: Electric - kWh, Instance: 1, Index: 0
2019-08-12 08:50:03.933 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 1: Electric - W, Instance: 1, Index: 2
2019-08-12 08:50:03.933 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 1: Exporting, Instance: 1, Index: 256
2019-08-12 08:50:03.976 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 2: Electric - kWh, Instance: 2, Index: 0
2019-08-12 08:50:03.976 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 2: Electric - W, Instance: 2, Index: 2
2019-08-12 08:50:03.976 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 2: Exporting, Instance: 2, Index: 256
2019-08-12 08:50:03.498 Status: OpenZWave: Received timeout notification from HomeID: 4100944663, NodeID: 13 (0x0d)
2019-08-12 08:50:04.019 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 3: Electric - kWh, Instance: 3, Index: 0
2019-08-12 08:50:04.019 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 3: Electric - W, Instance: 3, Index: 2
2019-08-12 08:50:04.019 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: METER, Label: Instance 3: Exporting, Instance: 3, Index: 256
2019-08-12 08:50:04.060 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: CENTRAL SCENE, Label: Scene 1, Instance: 1, Index: 1
2019-08-12 08:50:04.060 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: CENTRAL SCENE, Label: Scene 2, Instance: 1, Index: 2
2019-08-12 08:50:04.270 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 1: Previous Event Cleared, Instance: 1, Index: 256
2019-08-12 08:50:04.270 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 1: Power Management, Instance: 1, Index: 8
2019-08-12 08:50:04.311 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 1: Error Code, Instance: 1, Index: 266
2019-08-12 08:50:04.311 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 1: System, Instance: 1, Index: 9
2019-08-12 08:50:04.398 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 2: Previous Event Cleared, Instance: 2, Index: 256
2019-08-12 08:50:04.398 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 2: Power Management, Instance: 2, Index: 8
2019-08-12 08:50:04.441 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 2: Error Code, Instance: 2, Index: 266
2019-08-12 08:50:04.441 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 2: System, Instance: 2, Index: 9
2019-08-12 08:50:04.527 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 3: Previous Event Cleared, Instance: 3, Index: 256
2019-08-12 08:50:04.527 OpenZWave: Value_Added: Node: 13 (0x0d), CommandClass: ALARM, Label: Instance 3: Power Management, Instance: 3, Index: 8
2019-08-12 08:50:05.607 (ZwaveUSB) General/kWh (Badkamer_kWh Meter1)
2019-08-12 08:50:05.667 (ZwaveUSB) Usage (Badkamer_Unknown1)
2019-08-12 08:50:05.667 (ZwaveUSB) General/kWh (Badkamer_kWh Meter1)
2019-08-12 08:50:05.732 (ZwaveUSB) General/kWh (Badkamer_kWh Meter2)
2019-08-12 08:50:05.793 (ZwaveUSB) Usage (Badkamer_Unknown2)
2019-08-12 08:50:05.794 (ZwaveUSB) General/kWh (Badkamer_kWh Meter2)
2019-08-12 08:50:06.056 (ZwaveUSB) General/kWh (Badkamer_kWh Meter3)
2019-08-12 08:50:06.118 (ZwaveUSB) Usage (Badkamer_Unknown3)
2019-08-12 08:50:06.118 (ZwaveUSB) General/kWh (Badkamer_kWh Meter3)
2019-08-12 08:50:06.419 Status: OpenZWave: Awake Nodes queried

**Comment by petergebruers** [comment]
Thanks, very kind of you and interesting, but that log does not contain any useful information about this issue. The log does not have the level of detail I expect. If it would, it would be much, much larger and that is why I asked to zip and drop it here... Don't paste it in a text box...

Can you please enable debug logging on the Z-Wave controller, then do "Refresh node info" + change whatever config you are trying to change, then zip and upload OZW_Log.txt?

That file would contain hundreds or thousands of lines, quite a few with hexadecimal codes, like this

```
2019-08-12 09:07:13.766 Info, Node001, Sending (Command) message (Callback ID=0x00, Expected Reply=0x80) - Get Routing Info (Node=1): 0x01, 0x07, 0x00, 0x80, 0x01, 0x00, 0x00, 0x03, 0x7a
2019-08-12 09:07:13.766 Info, Node001, Encrypted Flag is 0
2019-08-12 09:07:13.774 Detail, Node001,   Received: 0x01, 0x20, 0x01, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x5e
```

**Comment by TerrorSource** [comment]
I did exactly that.
- Enabled DEBUG in general Domoticz settings.
- Enabled DEBUD in Aeotec Gen.5 stick config
- Refresh node info on the Walli Switch.

I can change a config on the Walli Switch but i cannot save it. I can press the "Apply configuration for this device" button but it does nothing. While i can change config's on non-Walli sensors.

**Comment by petergebruers** [comment]
I believe you, but you did not zip and drag & drop the OZW_Log.txt file here... That file is in your Domoticz folder, under Config

**Comment by TerrorSource** [comment]
Hopefully this is what you want/need.

[OZW_Log.zip](https://github.com/domoticz/domoticz/files/3491369/OZW_Log.zip)

**Comment by petergebruers** [comment]
Yes, excellent, that is the file! I had a quick look and I can see you've refreshed the node info, that's good. Detailed analysis will take more time.

**Comment by petergebruers** [comment]
I am pretty sure it is caused by the definition of parameters 30 to 34. They are actually not lists, but 32 bit integer values. I don't own this device so I cannot test this, but I can revise the config file so you can test it...

**Comment by petergebruers** [comment]
Here's the file. Please stop Domoticz, unzip, overwrite the existing file in your domoticz/config/fibaro folder with this one, start Domoticz and do "refresh node info" on this node (in hardware, openzwave panel of domoticz). Then please try to change a parameter and tell me what happens, and zip and drag & drop the OZW_Log.txt file here like you did previously. That was really helpful.

[fgwds221.xml.zip](https://github.com/domoticz/domoticz/files/3583432/fgwds221.xml.zip)

**Comment by lwuilmart** [comment]
hello, I had the same issue, not able to apply the configuration. I did what you mention and it's working now! thanks!

**Comment by petergebruers** [comment]
Thank you for chiming in! I'll push the change to OpenZWave master, probably today.

**Comment by TerrorSource** [comment]
@petergebruers Works!

Can you do the same for the Walli Dimmer? Which has the same issue.

**Comment by petergebruers** [comment]
Voilà. Definitions fixed in OpenZWave Version 1.6-912-gcc7a277c

**Comment by TerrorSource** [comment]
Fixed by @petergebruers
